padd.sh

```diff
@@ -1384,7 +1401,6 @@ DisplayHelp() {
 :::  -h, --help     display this help text
```

Comment:
You forgot to add the new option to the `DisplayHelp()` function.

Reply:
Thanks. I pushed an update.